### PR TITLE
Lesson 1.9

### PR DESCRIPTION
**Pull request by stephskas**
setting up Routes in App.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,47 +1,66 @@
...
-  const addTodo = (newTodo) => {
-      setTodoList([...todoList, newTodo]);
-  }
+	useEffect(() => {
+		const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`;
+		try {
+			fetch(url, {
+				headers: {
+					Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+				},
```
Comment (royemosby) [inline src/App.js:17]: If you ever find that directly applying values to an argument for a function like fetch, you can always create a configuration object and pass it in. For example:
```
const options = {
  method: 'GET',
  headers: {
    Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
  },
};
fetch(url, options)
```
This is a matter of style preference and what you have is completely fine.

Comment (stephskas) [reply src/App.js:17]: Thanks!

**Comment by royemosby** [comment]
I left some comments for 1.8 on Slack in case you don't check that as frequently as here.

**Comment by stephskas** [comment]
Thanks Roy!On Jan 26, 2023, at 2:35 PM, Roy Mosby ***@***.***> wrote:﻿
I left some comments for 1.8 on Slack in case you don't check that as frequently as here.

**Pull request merged**